src/content/h1/tools/aether/readme.md

```diff
@@ -1,15 +1,219 @@
+
+<figure>
+  <a href="Completed_Render_to_Texture.jpg">
+    <img src="Completed_Render_to_Texture.jpg" alt="Completed render to texture dialogue"/>
+  </a>
+  <figcaption>
+    <p>Completed render to texture dialogue</p>
+  </figcaption>
+</figure>
+
+The Output area is where you set up what bitmaps you want to create and how big you want them to be. Hit Add and select Lighting Map. You can leave everything default.
+
+Earlier, the reason you didn't import the lightmap meshes as a single object is because the bitmaps that are applied to them are of varying sizes and you can't set individual sizes for sub objects in the Render to Texture dialog. So, open up Aether again and open your bitmap for editing. This is just so we can see the current bitmap sizes being used for each lightmap. Next, in the Render to Texture dialog change the "All Selected" option to "Individual" so that we can change each individual object in our selection rather than editing them all at once. Near the top of the dialog there is a list with all of the currently selected meshes in it. Select the object at the top of the list. This is lightmap 0, so go to lightmap 0 in Aether and look at its size. For beavercreek the size is 256x128, but we want higher resolution bitmaps so quadrupling the bitmap size means we are creating a bitmap that is 1024x512. Therefore in MAX set the width and height of lightmap 0 to 1024 by 512. You will need to do this for all of the lightmap meshes.
+
+There is a limit to how much bitmap data a bitmap tag is permitted to hold so if your BSP has alot of lightmaps you may need to reduce your bitmap sizes to only double the original size to make them fit. If you are using an earlier version of 3ds MAX, you might only have a width value, so set that to the larger of the width or height values, as Aether can scale your bitmap when you import it. It's advised to use a minimum bitmap size of 128x128, because if you have a lot of bitmaps that are small you can set their size to 128x128 when "All Selected" is used and only have to manually change those that end up being bigger than 128x128.
+
+Finally set "Individual" back to "All Selected". Now you are all set up and ready to render your lightmaps. So hit render. Even if you set rectangular bitmap resolutions such as 1024x512, they will still render as a square but will be scaled before being saved. You will notice that the render preview will show it rendering with diffuse maps and such, but the output bitmaps will only have the lighting information. 
+
+**Aether - Importing Lightmap Bitmaps**
```

Comment:
Headings should be sentence case. The editor's guide has our writing style guide among other tips:

https://github.com/Sigmmma/c20/blob/master/EDITORS.md